docs/app/guides/cypress-studio.mdx

```diff
@@ -66,6 +76,26 @@ When Studio is enabled, you can:
 
 Supported action commands include: [`.click()`](/api/commands/click), [`.type()`](/api/commands/type), [`.check()`](/api/commands/check), [`.uncheck()`](/api/commands/uncheck), [`.select()`](/api/commands/select)
 
+## AI-Powered Recommendations
+
+Cypress Studio AI enhances your test creation workflow by automatically analyzing your interactions and suggesting relevant assertions in real time. As you record actions, the AI appears in the Studio Panel with intelligent recommendations tailored to each step.
+
+### How AI Recommendations Work
+
+1. **Automatic Analysis**: As you interact with elements, the AI analyzes the context and element properties
+2. **Smart Suggestions**: Relevant assertions appear in the Studio Panel based on your actions
+3. **Accept or Reject**: Review each recommendation and choose to accept or reject it
+4. **Seamless Integration**: Accepted recommendations can be saved to your test file
+
+### Types of AI Recommendations
+
+The AI Suggests assertions based on changes within your App after interactions occur.
+
+- **Visibility checks**: Ensure elements are visible or hidden
+- **Text validation**: Verify text content matches expectations
+- **Form validation**: Check input values, selections, and form states
+- **Element properties**: Validate attributes, classes, and other properties
+
```

Comment:
The `AI-Powered Recommendations` section was added inside the existing `How Cypress Studio Works` section and at the same hierarchy level.  Should this be moved ahead of the title of the `How Cypress Studio Works` section as a standalone section to describe what Studio AI is?  

Then in the `Add assertions` section, should it explain the two ways of adding assertions (manual vs AI) and when to use each one?